content/10.zk-stack/10.zk-chains.md

```diff
@@ -1,34 +1,34 @@
 ---
-title: ZK Chains
-description: "Delve into the concept of ZK chains and rollup clusters."
+title: ZKsync Chains
+description: "Delve into the concept of ZKsync chains and rollup clusters."
 ---
 
-ZK chains represent a sophisticated layer of blockchain architecture called a **rollup cluster**,
+ZKsync chains represent a sophisticated layer of blockchain architecture called a **rollup cluster**,
 consisting of parallel-running chains that achieve consensus and finality on Ethereum's Layer 1 (L1) through a shared bridge.
 
-ZK chains are fully interoperable within the Elastic Network, facilitating seamless interactions across chains.
+ZKsync chains are fully interoperable within the Elastic Network, facilitating seamless interactions across chains.
 
 ![elastic_network](/images/zk-stack/elastic_network.png)
 
-ZK chains operate with a shared bridge contract on Ethereum's L1 and include native bridges between individual rollups,
-enhancing the overall interoperability and efficiency of the network. Key features of ZK chains include:
+ZKsync chains operate with a shared bridge contract on Ethereum's L1 and include native bridges between individual rollups,
+enhancing the overall interoperability and efficiency of the network. Key features of ZKsync chains include:
 
-1. **Trustless Validating Bridges**: Ensures that rollups within the ZK chain are interconnected without requiring additional trust layers.
+1. **Trustless Validating Bridges**: Ensures that rollups within the ZKsync chain are interconnected without requiring additional trust layers.
 2. **Asset Transfers**: Interoperability simplifies the transfer of assets, including burning and minting mechanisms, across the ecosystem.
 3. **Unified Governance**: Leveraging a shared governance framework on L1,
   the ecosystem can coordinate updates or respond collectively to vulnerabilities, much like a traditional blockchain network would handle a fork.
-4. **Security and Trust**: All ZK chains must utilize the standardized zkEVM engine to maintain consistent security and operational standards,
+4. **Security and Trust**: All ZKsync chains must utilize the standardized zkEVM engine to maintain consistent security and operational standards,
```

Comment:
```suggestion
4. **Security and Trust**: All ZKsync chains must utilize the standardized ZK-engine to maintain consistent security and operational standards,
```
I think zkEVM is not the most accurate term here anymore. Maybe ZK-engine as what we need is all chains to use the same proof system.